workflow/executor/executor.go

```diff
@@ -40,13 +40,6 @@ import (
 	os_specific "github.com/argoproj/argo/workflow/executor/os-specific"
 )
 
-var MainContainerStartRetry = wait.Backoff{
```

Comment:
maybe no need to move this to a shared package which no one will share?

Reply:
sure will move it back :)